debian/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:sid
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -o Acquire::ForceIPv4=true update && \
+    apt-get -o Acquire::ForceIPv4=true install -yq \
```

Comment:
You need to set `DEBIAN_FRONTEND` for the "install" line, too.

Reply:
Fixed.